hknweb/forms.py

```diff
@@ -21,6 +25,32 @@ class SignupForm(UserCreationForm):
      last_name = forms.CharField(max_length=30, required=True)
      email = forms.EmailField(max_length=200, required=True)
 
+     def clean_email(self):
+         email = self.cleaned_data.get('email')
+         if (email == None or email.split('@')[-1] != 'berkeley.edu'
+         or email.split('@')[1] != email.split('@')[-1]):
+             raise forms.ValidationError('Please use your Berkeley email', code='invalid')
```

Comment:
I think everything looks good but do we only want to allow accounts with Berkeley e-mails?

Reply:
I talked to James and he was like yeah, this is the easiest solution rn